apps/client/src/common/stores/clientStore.ts

```diff
@@ -0,0 +1,49 @@
+import { create } from 'zustand';
+
+interface ClientStore {
+  self?: string;
```

Comment:
i think self is a loaded word in software and we should not use it here. name, setName seem to me like reasonable choices for the feature

Reply:
hahaha, i guess it was likely me then :D 

self usually means the instance. in javascript, it was common to assign this to self to keep the value of this across scopes

in python self is also the reference to the instance

what I meant is that self tend to mean things in programming, none of which are the string reference of a connection